.github/workflows/CI.yml

```
@@ -3,6 +3,7 @@ name: CI
 on:
   pull_request:
     types: [opened, synchronize]
+    branches: [develop, releases/**]
```

```suggestion
    branches: [develop, release/**]
```

我们的 release 分支没有 `s`